README.md

```diff
@@ -25,6 +56,166 @@ TAF package consists of two folders: ./taf-repo and ./testcases-repo. taf-repo f
+ ```
+     [Service]
+     Environment="HTTP_PROXY=http://<proxy server>:<port>/"
+ ```
+
+- If you have internal Docker registries that you need to contact without proxying you can specify them via the NO_PROXY environment variable:
+```
+      [Service]
+      Environment="HTTP_PROXY=http://<proxy server>:<port>/"
+      Environment="NO_PROXY=localhost,127.0.0.1,.<example.com>"
+```
+
+- Flush changes:
+```
+      $ root@ 16-04:~# systemctl daemon-reload
+```
+
+- Verify that the configuration has been loaded:
+```
+      $ root@ 16-04:~# systemctl show --property Environment docker
+        Environment=HTTP_PROXY=http://proxy.example.com:80/export
+```
+
+- Restart Docker:
+```
+      $ root@ 16-04:~# service docker restart
+```
+
+**Settings**
+
+If your environment requires specific proxy settings, perform the next steps accordingly:
+
+- Update or add proxy settings in Dockerfile for building **TAF** Docker image. Below are specified proxy settings which Dockerfile uses to build TAF image, by default:
+```
+        ENV http_proxy 'http://<proxy server>:<port>'
+        ENV https_proxy 'http://<proxy server>:<port>'
+        ENV ftp_proxy='http://<proxy server>:<port>'
+        ENV socks_proxy='socks://<proxy server>:<port>'
+        ENV no_proxy='localhost,127.0.0.1,.<example.com>'
+```
+
+- Update or specify environment variables in file ‘docker_environment_variables’ that should be run inside container. This file is located in the root of the **TAF** repository.
+
+**Build image**
+
+To build **TAF** image you need to execute command:
+```
+    docker build -t <image_name>:<tag> -f Dockerfile .
+    E.g. :root@14-04:~/taf# docker build -t ubuntu:taf3 -f Dockerfile .
+```
+
+Building of **TAF** image from a Dockerfile can take over 15 minutes.
+
+**Run container**
+
+To run the **TAF** image and login to the container, execute command:
+```
+    docker run -i -t  <image_name>:<tag>
+    E.g.: root@14-04:~# docker run -i -t ubuntu:taf3
+```
+
+After execution of previous command, the prompt should changed to:
+```
+    root@<container_id>:~#
+    E.g.: root@fa3ced1fdcbf:~#
+```
+
+The **TAF** repo can be found in the root directory. Once in the **TAF** container, the sample testcases can be run:
+```
+    $ cd unittests
+    $ /root/taf/unittests/ py.test test_common3.py
+```
+
+**Start container that is in exit state**
+
+If container is in exit status, restart and reattach via commands:
+```
+    docker start <container_id>
+    docker attach <container_id>
+    E.g.: $ root@14-04:~# docker start fa3ced1fdcbf
+          $ root@14-04:~# docker attach fa3ced1fdcbf
+```
+**Useful commands:**
+
+- Show all images:
+
+    `docker images`
+- See running containers:
+
+    `docker ps`
+- See all containers and their status:
+
+    `docker ps -a`
+
+
+### Simple test execution
+
+> (TODO: Create pull request in test repo with simple localhost config and linux_host test case)
```

Comment:
@lijenni Observed that you correct the project name, so we'd better clarify the name of the project. How about to use TAF as project name, and taf3 as a version which supports Python 3?